clusters/build-clusters/01_cluster/sshd-bastion/coredns/coredns_config.yaml

```diff
@@ -8,11 +8,11 @@ data:
             rewrite stop name exact lnxocp01 sshd-0.bastion-z.svc.cluster.local
             rewrite stop name exact lnxocp02 sshd-1.bastion-z.svc.cluster.local
             rewrite stop name exact lnxocp03 sshd-2.bastion-z.svc.cluster.local
-            rewrite stop name regex ^.*libvirt-s390x-([0-9])-[0-9]\.$ sshd-{1}.bastion-z.svc.cluster.local
+            rewrite stop name regex ^.*libvirt-s390x-([0-9])-[0-9].* sshd-{1}.bastion-z.svc.cluster.local
```

Comment:
We might want to remove `^.*libvirt-s390x-([0-9])-[0-9]*`, since the request URL will be `LEASED_RESOURCE-JOB_HASH`

Reply:
The `.` is part of the regex, not a literal dot. It means any character. Coupled with `*`, it translates to any character 0 or more times. I've replaced the literal dot `\.` we had before. `[0-9]*` means a digit 0 or more times.